### PR TITLE
feat:added LND connect URL for REST API

### LINKED ISSUE #478: Feature Request: add LND connect URL for REST API.
**Is your feature request related to a problem? Please describe.**
LND connect Url contains parameters to work with GRPC API and not REST.

**Describe the solution you'd like**
Add a new field and a copy button for LND Connect REST API.

**Additional context**
image

### PR DESCRIPTION
**Pull request by NehaK745**
Closes #478

### Description

added URL for REST API

### Screenshots
Polar 17-08-2021 05_33_11 PM (2)

### CODE REVIEW

File: src/components/designer/lightning/connect/LndConnect.tsx [hunk 1]
```diff
@@ -21,16 +21,28 @@ const LndConnect: React.FC<Props> = ({ node }) => {
...
       const adminMac = await read(adminMacaroon, 'hex');
       const invoiceMac = await read(invoiceMacaroon, 'hex');
       const readonlyMac = await read(readonlyMacaroon, 'hex');
 
       const values: Record<string, string> = {};
-      values[l('adminUrl')] = encode({ host, cert, macaroon: adminMac });
-      values[l('invoiceUrl')] = encode({ host, cert, macaroon: invoiceMac });
-      values[l('readOnlyUrl')] = encode({ host, cert, macaroon: readonlyMac });
-
+      values[l('GRPCadminUrl')] = encode({ host, cert, macaroon: adminMac });
```
Comment (jamaljsr) [inline src/components/designer/lightning/connect/LndConnect.tsx:32]: Just a small nitpick, can you change these keys to camel case. ex: `grpcAdminUrl`, `restInvoiceUrl`, etc.

**Review by jamaljsr — CHANGES_REQUESTED** [review]
This looks good, I just requested a very minor change. Otherwise, great job!

File: src/components/designer/lightning/connect/LndConnect.tsx (hunk 1 above)
Comment (NehaK745) [reply src/components/designer/lightning/connect/LndConnect.tsx:32]: ok I will do that

**Review by bolatovumar — APPROVED** [review]

**Review by jamaljsr — APPROVED** [review]
LGTM

**Pull request merged**